editor/src/components/inspector/sections/component-section/cartouche-ui.tsx

```diff
@@ -129,13 +129,16 @@ export const CartoucheUI = React.forwardRef(
                 textOverflow: 'ellipsis',
 
                 /* Beginning of string */
-                direction: source === 'literal' ? 'ltr' : 'rtl', // TODO we need a better way to ellipsize the beginnign because rtl eats ' " marks
+                // commented it because it messes with the order of children
```

Comment:
we must fix this before merging, otherwise the cartouche ellipsization is broken

Reply:
fixed here https://github.com/concrete-utopia/utopia/pull/5852/commits/8c697b53eaf013a1e4de0eaadb46a84a5da0e691